docs/src/api/class-apiresponseassertions.md

```diff
@@ -74,7 +73,7 @@ The opposite of [`method: APIResponseAssertions.toBeOK`].
 * langs:
   - alias-java: isOK
 
-Ensures the response status code is within [200..299] range.
+Ensures the response status code is within `200..299` range.
```

Comment:
this ended up breaking the dotnet docs xml generator, so simplifying it.

Reply:
nit: can it be `[200..299]` ?